examples/todo-intermediate/test/unit/controllers/todo-list.controller.unit.ts

```diff
@@ -0,0 +1,165 @@
+import {expect, sinon} from '@loopback/testlab';
+import {TodoListController} from '../../../src/controllers';
+import {TodoList} from '../../../src/models/index';
+import {TodoListRepository} from '../../../src/repositories';
+import {givenTodoList} from '../../helpers';
+
+describe('TodoController', () => {
+  let todoListRepo: TodoListRepository;
+
+  /*
+  =============================================================================
+  METHOD STUBS
+  These handles give us a quick way to fake the response of our repository
+  without needing to wrangle fake repository objects or manage real ones
+  in our tests themselves.
+  =============================================================================
+   */
+  let create: sinon.SinonStub;
+  let count: sinon.SinonStub;
+  let find: sinon.SinonStub;
+  let updateAll: sinon.SinonStub;
+  let deleteAll: sinon.SinonStub;
+  let findById: sinon.SinonStub;
+  let updateById: sinon.SinonStub;
+  let deleteById: sinon.SinonStub;
```

Comment:
These stubs seem to be repeated in multiple test files. Is it time to refactor/extract out a shared helper for setting up a set of stubs for CrudRepository?

Reply:
Considering that these stub references are just references and the actually setting up of the stubs is done by a single line (`createStubInstance(ctor)`), I think we can just refactor the code to avoid defining these 'shortcut' references and just use direct references to the stub methods in our tests. Thoughts?